packages/core/js-client/src/jsPeer/__test__/peer.spec.ts

```diff
@@ -72,17 +72,17 @@ describe('FluencePeer usage test suite', () => {
 
     it('Should not crash if undefined is passed as a variable', async () => {
         await withPeer(async (peer) => {
-            const res = await new Promise<any>((resolve, reject) => {
-                const script = `
+            const script = `
         (seq
             (call %init_peer_id% ("load" "arg") [] arg)
             (seq
                 (call %init_peer_id% ("op" "identity") [arg] res)
                 (call %init_peer_id% ("callback" "callback") [res])
             )
         )`;
-                const particle = peer.internals.createNewParticle(script);
-
+            const particle = await peer.internals.createNewParticle(script);
+            
+            const res = await new Promise<any>((resolve, reject) => {
```

Comment:
I hope some day in this repo `any` would be completely forbidden to use
There is always a way to not use `any`

Reply:
Yeap, quite soon